integration/kubernetes/helm-chart/alluxio/templates/config/alluxio-conf.yaml

```diff
@@ -44,5 +44,5 @@ data:
   ALLUXIO_FUSE_JAVA_OPTS: |-
     -Dalluxio.user.hostname=${ALLUXIO_CLIENT_HOSTNAME} -Dalluxio.worker.hostname=${ALLUXIO_CLIENT_HOSTNAME} {{ .Values.fuse.jvmOptions }} {{- range $key, $val := .Values.fuse.properties }} -D{{ $key }}={{ $val }} {{ end }}
   ALLUXIO_USER_JAVA_OPTS: |-
-    -Dalluxio.master.hostname=${ALLUXIO_MASTER_HOSTNAME}
+    {{ if $isHaEmbedded }}-Dalluxio.master.hostname=${ALLUXIO_MASTER_HOSTNAME}{{end}}
```

Comment:
I believe this is only used for HA

Reply:
neither actually. `ALLUXIO_JAVA_OPTS` should suffice.

Reply:
can you test if we can remove this? thx

Reply:
I see. `ALLUXIO_USER_JAVA_OPTS` has been removed and validated by tests.